homeassistant/components/upnp/binary_sensor.py

```diff
@@ -28,7 +28,6 @@ class UpnpBinarySensorEntityDescription(
 SENSOR_DESCRIPTIONS: tuple[UpnpBinarySensorEntityDescription, ...] = (
     UpnpBinarySensorEntityDescription(
         key=WAN_STATUS,
-        name="wan status",
```

Comment:
Is it intentional to not have a custom name for the binary sensor? Is it clear that a binary sensor named "Connectivity" reflects WAN status?

Reply:
Or `Connected`, or `Active`?

Reply:
The change I made (it was more a proposal) is using device class translations. By using connectivity it should be in line with other integrations providing a binary sensor with device class connectivity. The only question now is if this "dekt de lading" correctly.